### DISCUSSION TITLE
s2-vue 是否支持  gridAnalysis: 网格分析表 strategy: 趋势分析表

### DISCUSSION THREAD

**Discussion by chanbaihai** [antvis/S2#2303] [Q&A]
我看官方文档写的 react组件支持四种
![image](https://github.com/antvis/S2/assets/36693835/b2597f95-55d5-492d-9b99-995f43b2f449)
 vue的只写了两种
![image](https://github.com/antvis/S2/assets/36693835/50853391-68a1-4d44-9be2-29606c775168)
官方文档地址 https://s2.antv.vision/api/components/sheet-component#props
gridAnalysis示例地址 https://s2.antv.antgroup.com/examples/case/proportion/#group-drill-down
我拿gridAnalysis示例数据在s2-vue的包下面，没有渲染出来对应的效果
预期效果
![image](https://github.com/antvis/S2/assets/36693835/383cebad-7a49-4e64-b260-577545486f98)
实际结果
![image](https://github.com/antvis/S2/assets/36693835/80c712fe-d0d3-451d-9a21-6473f7a8c9d8)

要使用剩下两种类型的表格该如何操作？

**Comment by lijinke666** [comment]
> 我拿gridAnalysis示例数据在s2-vue的包下面，没有渲染出来对应的效果

网格分析表/趋势分析表 都是通过 `自定义单元格` 实现的, 如果你想在 Vue 中使用, 参考下 React 的实现自己封装下即可, 或者直接 PR

> 趋势分析表

https://github.com/antvis/S2/blob/807648180fe64f47e9f70df865890ab480b07603/packages/s2-react/src/components/sheets/strategy-sheet/index.tsx#L82-L89

> 网格分析表

https://github.com/antvis/S2/blob/807648180fe64f47e9f70df865890ab480b07603/packages/s2-react/src/components/sheets/grid-analysis-sheet/index.tsx#L13-L16
